src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,242 @@
+import React, { forwardRef, useRef } from "react";
+import PropTypes from "prop-types";
+import { COLORS_BASIC, SIZES_BASIC } from "../../constants";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import "./Slider.scss";
```

Comment:
please use css modules while implement new components instead of bem (this is a new convention and therefore we it is hard to find references)

Reply:
For a while I've moved BEM helper inside component itself (from commons reusable helpers).
(https://github.com/mondaycom/monday-ui-react-core/pull/424/commits/441ef12a7478e650c4cf431724c6b8ad3d6acc9a)
When I'll finished with other sub-tasks/issues I'll start work on switch to CSS Modules.

Reply:
According to our agreements move it to Backlog: Slider Improvements tasks